src/About/SupportCard.tsx

```diff
@@ -52,6 +70,39 @@ export default () => {
                     label="Create system report"
                 />
             </Section>
+            <Section title="Verbose Logging">
+                <p>
+                    Aid our support team with additional log information. Enable
+                    this only when necessary as the log will grow quickly.
```

Comment:
Small enhancement request, but maybe this text: `Enable this only when necessary as the log will grow quickly.` should be colored slightly different to make it stand out a bit as it is important that the users see this.